docs/admin/release_notes/version_3.0.md

```diff
@@ -0,0 +1,66 @@
+
+**Device Lifecycle Management App** version 3.0 now supports these core Nautobot models: SoftwareVersion, SoftwareImageFile, and Contact.
+
+These models were introduced in Nautobot 2.2.0 as functional equivalents to the now deprecated DLM models. **DLM models will be removed in version 4.2**. The table below shows the corresponding model in core that matches each DLM model.
+
+| Core model | DLM model |
+| :---- | :---- |
+| SoftwareVersion | SoftwareLCM |
+| SoftwareImageFile | SoftwareImageLCM |
+| Contact | ContactLCM |
+
+The DLM models and their instances will remain in place to ensure a smooth migration and prevent data loss.
+
+!!! note
+    After installing app version 3.0, all existing instances of these models must be moved to the core models. This will enable full DLM app functionality. The migration process is outlined in the [Migrating to DLM app version 3.0](../migrating_to_v3.md) guide.
+
+!!! warning
+    Ensure that the DLM app is at least version 3.0.0 and Nautobot is version 2.2.0 or later before starting migration.
+
+
+### Added
+
+#### Device Hardware Notice Reporting
+
+A new reporting type, **Device Hardware Notice Report**, has been introduced. This report identifies affected device types and the quantity of device instances impacted by hardware notices. This functionality mirrors the existing validated software reporting.
+
+### Device View with Contracts
+
+The device detail page will display up to five contracts, sorted by end date (latest first). Inventory item detail pages will also display contract information for associated items.
+
+<!-- towncrier release notes start -->
+
+## [v3.0.0 (2025-03-14)](https://github.com/nautobot/nautobot-app-device-lifecycle-mgmt/releases/tag/v3.0.0)
+
+### Added
+
+- [#404](https://github.com/nautobot/nautobot-app-device-lifecycle-mgmt/issues/404) - Added a note to the placeholder software image files for Nautobot v2.2.0-v2.3.0.
+- [#405](https://github.com/nautobot/nautobot-app-device-lifecycle-mgmt/issues/405) - Added a banner to all device lifecycle management views if any of the SoftwareLCM, SoftwareImageLCM or ContactLCM models have not been migrated to core models.
+
+### Changed
+
+- [#261](https://github.com/nautobot/nautobot-app-device-lifecycle-mgmt/issues/261) - Updated template extensions to align with standard Nautobot UI views.
+- [#403](https://github.com/nautobot/nautobot-app-device-lifecycle-mgmt/issues/403) - Updated the migration job to duplicate relationships instead of directly migrating them.
+- [#404](https://github.com/nautobot/nautobot-app-device-lifecycle-mgmt/issues/404) - Updated the DLM to core migration job to only create placeholder software image files for Nautoobot v2.2.0-v2.3.0.
+- [#404](https://github.com/nautobot/nautobot-app-device-lifecycle-mgmt/issues/404) - Changed the post_migrate_create_relationships signal to hide the device_soft and inventory_item_soft relationships to prevent 500 errors on Device and InventoryItem urls.
```

Comment:
We can probably remove these since they were changing features that were never released.